src/sentry/issues/occurrence_consumer.py

```diff
@@ -43,6 +52,92 @@ def create_ingest_occurences_consumer(
     )
 
 
+def save_event_from_occurrence(
+    data: Dict[str, Any],
+    **kwargs: Any,
+) -> Event:
+
+    from sentry.event_manager import EventManager
+
+    event_type = "platform"
+
+    with metrics.global_tags(event_type=event_type):
+        project_id = data["data"].pop("project_id")
+        data = CanonicalKeyDict(data)
+
+        with metrics.timer("occurrence_consumer.save_event_occurrence.event_manager.save"):
+            manager = EventManager(data)
+            event = manager.save(project_id=project_id)
```

Comment:
At the moment this will run through saving the event either as an error or a transaction. We need another branch in eventmanager for our new events. I'm not sure if this also means you'll need to add a new event type to `eventtypes/`

Reply:
ok. will investigate